js/index.js

```diff
@@ -1 +1,73 @@
 // Your code goes here
+
+
+const letsGoParagraph = document.querySelector(".text-content p");
+letsGoParagraph.addEventListener('click', function(event){
+    letsGoParagraph.textContent = "new words"
+})
+
+//MOUSEOVER
+letsGoParagraph.addEventListener('mouseover', function(event){
+    letsGoParagraph.style.color = "red"
+})
+
+//WHEEL
+const firstH2 = document.querySelector(".intro h2");
+firstH2.addEventListener('wheel', function(event){
+    firstH2.style.color = "blue"
```

Comment:
If you're not using the `event` the event listener is passing into your call back function, you can just omit the `event` parameter.